src/guiguts/application.py

```diff
@@ -586,6 +590,7 @@ def init_help_menu(self) -> None:
         menu_help = Menu(menubar(), "~Help")
         menu_help.add_button("Guiguts ~Manual", self.show_help_manual)
         menu_help.add_button("About ~Guiguts", self.help_about)
+        menu_help.add_button("~Compose Sequences", ComposeHelpDialog.show_dialog)
```

Comment:
Two things --
* Should the menu item be changed to match the title bar on the widget?
* This has the same `~` shortcut as line 570, above. Is that okay, since they're in different menus, or does it need to be changed?

Reply:
> * Should the menu item be changed to match the title bar on the widget?

I've changed the name, though I don't think they always  have to be the same, since the menu also gives context. This menu item was Help->Compose Sequences, and is now Help->List of Compose Sequences

> * This has the same `~` shortcut as line 570, above. Is that okay, since they're in different menus, or does it need to be changed?

That's okay - as long as they are in different menus, they can have the same shortcut. You would do `Alt+H C` for this help dialog and `Alt+T U C` for the Compose Dialog itself.

🤣 - as I was checking the accuracy of the above sentence, I did come across a menu keystroke clash: `Tools->~Unmatched` and `Tools->~Unicode`. I've change the first of those to `Un~matched`